src/adapters/postgres/fields-adapter.ts

```
@@ -1730,18 +1724,19 @@ export class PostgresFieldsService implements IServicelocatorfields {
     result = result.map(async (data) => {
       const originalValue = data.value;
       let processedValue = data.value;
+      console.log('ssss', originalValue, processedValue);
```

ESLint

[error] Replace `'ssss'` with `"ssss"`

(prettier/prettier)